### ISSUE TITLE
accessibility, not usable with talkback (android)

### ISSUE THREAD

**Issue by Sebseb01** [tuskyapp/Tusky#1187]
With android and talkback, all butons (reply, boost, like, ...) and profil are not accessible ! 

on time line it's really cool to can switch  tooths with next and prev control, but when open a tooth i needs buttons

on v5 all butons are accessible with talkback.

.

].

* * * *
- Tusky Version:6.0 
- Android Version:
- Android Device: Nokia 7.1
- Mastodon instance (if applicable):
N/A
- [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate. yes yes

**Comment by charlag** [comment]
Hey
These actions are now available as accessibilty actions. It's up and right gesture if I remember correctly. TalkBack should announce the actions.

**Comment by Sebseb01** [comment]
It's works, sorry. 

It's the first time, i sae an app it using this.

perfect ! 

and thanks for fast answer

**Comment by charlag** [comment]
No problem, I'm really excited that people are using this!
